nixos/modules/misc/documentation/test.nix

```diff
@@ -1,7 +1,7 @@
-{ nixosLib, pkgsModule, runCommand }:
+{ evalModules, pkgsModule, runCommand }:
 
 let
-  sys = nixosLib.evalModules rec {
+  sys = evalModules rec {
```

Comment:
Chaged, because I made _experimental_ explicit via  `nixosLibExpermiental`. Howerver, this particular test shouldn't care, so we access the experimental target directly.